tests/ci-cluster/gke.ts

```diff
@@ -55,6 +55,7 @@ export class GkeCluster extends pulumi.ComponentResource {
               enabled: true,
               provider: "CALICO",
             },
+            deletionProtection: false,
```

Comment:
Curious why this was set previously? I'm assuming there was a reason, otherwise wouldn't the tests always have failed?